src/Auction.hs

```diff
@@ -73,7 +74,13 @@ define = modifyDealerProg addDefn
 
 
 makeCall :: T.Call -> Action
-makeCall call = modify $ first (>- call)
+makeCall call = modify $ first (>- T.CompleteCall call T.Unalerted)
+
+makeAlertableCall :: T.Call -> String -> Action
+makeAlertableCall call alert = do
+    (bidding, dealerProg) <- get
+    let bidder = currentBidder bidding
+    put (bidding >- T.CompleteCall call (T.alertFor bidder alert), dealerProg)
```

Comment:
This feels like the wrong place to call `alertFor`. Could that be wrapped inside `CompleteCall instead? and maybe from there, there's some way to achieve this without `do` notation and the `get`, though I don't yet see it.

Reply:
Not obviously within `CompleteCall`, which has no sense of which direction is making the call. but arguably we only need `alertFor` when outputting LaTeX/etc., and this should go way far back there instead. Maybe `CompleteCall` is just a pair of a `Call` and a `Maybe String` after all...